homeassistant/config_entries.py

```diff
@@ -1481,6 +1481,19 @@ async def async_finish_flow(
                 result["handler"], flow.unique_id
             )
 
+        if existing_entry is not None and flow.handler != "mobile_app":
+            # This causes the old entry to be removed and replaced when it
+            # should most likely update the previous entry and abort the flow
+            # see https://developers.home-assistant.io/blog/2024/11/22/config-flow-unique-id/
+            report_usage(
+                "creates a config entry when another entry with the same unique ID "
+                "exists",
+                core_behavior=ReportBehavior.LOG,
+                core_integration_behavior=ReportBehavior.LOG,
+                custom_integration_behavior=ReportBehavior.LOG,
```

Comment:
Shuold we start with just logging core integrations?

Reply:
I'm not sure
I would start with logging all, and we can easily adjust later if it generates too much noise